src/TrackPerfHistProc.cxx

```diff
@@ -54,23 +54,24 @@ void TrackPerfHistProc::init() {
   // Create ROOT histograms, with location setup by the above factory
   tree->mkdir("all");
   tree->cd("all");
-  _allTracks = std::make_shared<TrackPerf::TrackHists>();
-  _allTruths = std::make_shared<TrackPerf::TruthHists>();
+  _allTracks = std::make_shared<TrackPerf::TrackHists>(false);
+  _allTruths = std::make_shared<TrackPerf::TruthHists>(false);
   h_number_of_tracks = new TH1F(
       "number_of_tracks", ";Number of seeds/tracks;Events", 100, 0, 300000);
   tree->mkdir("../real");
   tree->cd("../real");
-  _realTracks = std::make_shared<TrackPerf::TrackHists>();
-  _realTruths = std::make_shared<TrackPerf::TruthHists>();
+  _realTracks = std::make_shared<TrackPerf::TrackHists>(false);
+  _realTruths = std::make_shared<TrackPerf::TruthHists>(true);
   _realReso = std::make_shared<TrackPerf::ResoHists>();
   tree->mkdir("../fake");
   tree->cd("../fake");
-  _fakeTracks = std::make_shared<TrackPerf::TrackHists>();
+  _fakeTracks = std::make_shared<TrackPerf::TrackHists>(true);
   h_number_of_fakes = new TH1F("number_of_fakes",
                                ";Number of fake tracks;Events", 100, 0, 300000);
+  h_count = new TH1F("Count","C;C",20,0,10);
```

Comment:
What's this for?

Reply:
Ah sorry about that, this was from debugging an issue in the Gaudi version. Removed now!